### PR TITLE
Fix broken require

### PR DESCRIPTION
**Pull request by benkaiser**
When removing default exports this line was missed.

### CODE REVIEW

File: core-build/gulp-core-build-serve/src/UntrustCertTask.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@ export class UntrustCertTask extends GulpTask<void> {
 
   public executeTask(gulp: typeof Gulp, completeCallback: (error?: string) => void): void {
     /* tslint:disable */
-    const CertificateStore = require('./CertificateStore').default;
+    const CertificateStore = require('./CertificateStore').CertificateStore;
```
Comment (iclanton) [inline core-build/gulp-core-build-serve/src/UntrustCertTask.ts]: Can we just change these to imports?

**Comment by iclanton** [comment]
You'll need to run `rush change`

**Comment by benkaiser** [comment]
> You'll need to run `rush change`

Ah, was hoping the web editor would be enough

**Comment by iclanton** [comment]
If you want to do it just in the browser, drop a file in `common/changes/@microsoft/gulp-core-build-serve/benkaiser-fix-require_2019-03-04-00-00.json` with these contents:

```JSON
{
  "changes": [
    {
      "comment": "<DESCRIBE THE CHANGE HERE>",
      "packageName": "@microsoft/gulp-core-build-serve",
      "type": "patch"
    }
  ],
  "packageName": "@microsoft/gulp-core-build-serve",
  "email": "benkaiser@users.noreply.github.com"
}

```

File: core-build/gulp-core-build-serve/src/UntrustCertTask.ts (hunk 1 above)
Comment (benkaiser) [reply core-build/gulp-core-build-serve/src/UntrustCertTask.ts]: Done!

**Comment by benkaiser** [comment]
Resolved and updated

**Pull request merged**